packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-wrapper.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import { Button, Modal } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { getAuthenticationStatusInfo, isMobile, Jurisdiction } from '@deriv/shared';
+import { connect } from '../../Stores/connect';
+import RootStore from '../../Stores/index';
+import JurisdictionModalContent from './jurisdiction-modal-content';
+import { TJurisdictionCardType, TJurisdictionModalContentWrapperProps } from '../props.types';
+import JurisdictionCheckBox from './jurisdiction-modal-checkbox';
+import JurisdictionModalFootNote from './jurisdiction-modal-foot-note';
```

Comment:
Lets sort these import statements as per this guide https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules